Makefile

```diff
@@ -16,7 +16,7 @@ ifeq "$(USE_POETRY)" "true"
 	install-cmd := poetry install
 	run-cmd := poetry run
 else
-	install-cmd := pipenv sync
+	install-cmd := pipenv verify && pipenv sync
```

Comment:
I'm unsure if the `&&` here will result in `pipenv sync` silently not being run if the verify fails?

Reply:
The make command prints and errors if the verify fails:
```
...
Pipfile.lock is out-of-date. Run $ pipenv lock to update.
make: *** [Makefile:64: run-api-server] Error 1
```

Which seems reasonable to me